tests/integration/sliders/test_sliders.py

```diff
@@ -162,6 +162,71 @@ def test_slsl005_slider_tooltip(dash_dcc):
     )
 
 
+def test_slsl005_rangeslider_tooltip(dash_dcc):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        [
+            html.Div(
+                [
+                    html.Div(
+                        dcc.RangeSlider(
+                            min=0,
+                            max=100,
+                            value=[0, 65],
+                            tooltip={"always_visible": True, "placement": "top"},
+                        ),
+                        style=dict(height=100),
+                    ),
+                    html.Div(
+                        dcc.RangeSlider(
+                            min=0,
+                            max=100,
+                            value=[0, 65],
+                            tooltip={"always_visible": True, "placement": "top"},
+                        ),
+                        style=dict(height=100),
+                    ),
+                    html.Div(
+                        dcc.RangeSlider(
+                            min=0,
+                            max=100,
+                            value=[0, 65],
+                            tooltip={"always_visible": True, "placement": "top"},
+                        ),
+                        style=dict(height=100),
+                    ),
+                    html.Div(
+                        dcc.RangeSlider(
+                            min=0,
+                            max=100,
+                            value=[0, 65],
+                            tooltip={"always_visible": True, "placement": "top"},
+                        ),
+                        style=dict(height=100),
+                    ),
+                    html.Div(
+                        dcc.RangeSlider(
+                            id="test-slider",
+                            min=0,
+                            max=100,
+                            value=[0, 65],
+                            tooltip={"always_visible": True, "placement": "top"},
+                        ),
+                        style=dict(height=100),
+                    ),
+                ],
+                style=dict(maxHeight=300, overflowX="scroll"),
+            )
+        ]
+    )
+
+    dash_dcc.start_server(app)
+    dash_dcc.wait_for_element("#test-slider")
+    dash_dcc.percy_snapshot(
+        "slider-make sure tooltips are only visible if parent slider is visible"
```

Comment:
Oh right - this line is failing because percy snapshot names need to be unique, this matches https://github.com/plotly/dash-core-components/pull/878

Normally it's nice to keep these snapshot names shorter too, though of course I should have provided that feedback in https://github.com/plotly/dash-core-components/pull/878 😉